v3/plugins/ms-kubernetes-tools/vscode-kubernetes-tools/1.0.0/meta.yaml

```diff
@@ -13,5 +13,6 @@ firstPublicationDate: "2019-05-15"
 spec:
   containers:
     - image: "eclipse/che-remote-plugin-kubernetes-tooling-1.0.0:next"
```

Comment:
As far as I understand this image can be used for multiple vscode extensions, do you know what will be memory limit for the container if multiple vs code extensions define different memory limit?

Reply:
> As far as I understand this image can be used for multiple vscode extensions

I don't know any except the K8s Plugin by now.

> do you know what will be memory limit for the container if multiple vs code extensions define different memory limit?

 I don't. Do you?
Good question BTW )

Reply:
> I don't know any except the K8s Plugin by now.

Ah, I see that it's a special docker image for k8s tooling. 

> I don't. Do you?

I don't.